### DISCUSSION TITLE
Collection option to preview rows

### DISCUSSION THREAD

**Discussion by ztob** [directus/directus#9459] [Archive]
Hi,

We use directus a lot in our projects and thank you guys for effort you do every day!

Do you plan to implement some option to allow to preview a row from collection when click on it (with modal) rather than going to details page.
When you want to navigate through a lot of items it takes a lot of time and the experience is not the best when going to details page.
Also using arrow keys to navigate to the next element or pagination when a rows is open it would save a lot of time.

Waiting for feedback.

Thx.

**Comment by benhaynes** [comment]
Yup! Let's keep that conversation going over here: https://github.com/directus/directus/discussions/3843
